lib/OpenQA/Task/Asset/Limit.pm

```diff
@@ -54,23 +62,25 @@ sub _limit {
     my $dbh        = $app->schema->storage->dbh;
     my $update_sth = $dbh->prepare('UPDATE assets SET last_use_job_id = ? WHERE id = ?');
 
-    # remove all assets older than a certain number of days which do not belong to a job group
+    # remove all assets older than a certain duration which do not belong to a job group
     my $untracked_assets_storage_duration
-      = $OpenQA::Utils::app->config->{misc_limits}->{untracked_assets_storage_duration};
+      = $OpenQA::Utils::app->config->{misc_limits}->{untracked_assets_storage_duration} * 24;
     my $now = DateTime->now();
     for my $asset (@$assets) {
         $update_sth->execute($asset->{max_job} && $asset->{max_job} >= 0 ? $asset->{max_job} : undef, $asset->{id});
         next if $asset->{fixed} || scalar(keys %{$asset->{groups}}) > 0;
 
-        my $age = $now->delta_days(DateTime::Format::Pg->parse_datetime($asset->{t_created}))->in_units('days');
+        my $age        = $now->delta_days(DateTime::Format::Pg->parse_datetime($asset->{t_created}))->in_units('hours');
+        my $asset_name = $asset->{name};
         if ($age >= $untracked_assets_storage_duration || !$asset->{size}) {
-            _remove_if($app->db, $asset);
+            _remove_if($app->db, $asset,
+"Removing asset $asset_name (not in any group, age ($age hours) exceeds limit ($untracked_assets_storage_duration hours)"
+            );
         }
         else {
-            my $asset_name     = $asset->{name};
-            my $remaining_days = $untracked_assets_storage_duration - $age;
+            my $remaining_hours = $untracked_assets_storage_duration - $age;
             OpenQA::Utils::log_warning(
-                "Asset $asset_name is not in any job group, will delete in $remaining_days days");
+                "Asset $asset_name is not in any job group, will delete in $remaining_hours hours");
```

Comment:
for easier parsing larger values, guess we better stay with remaining_days as a fractional number

Reply:
ok